ci/gpu/build.sh

```diff
@@ -46,7 +46,7 @@ conda install "rmm=$MINOR_VERSION.*" "nvstrings=$MINOR_VERSION.*" "cudatoolkit=$
               "dask>=2.1.0" "distributed>=2.1.0" "numpy>=1.16" "double-conversion" \
               "rapidjson" "flatbuffers" "boost-cpp" "fsspec>=0.3.3" "dlpack" \
               "feather-format" "cupy>=6.0.0" "arrow-cpp=0.14.1" "pyarrow=0.14.1" \
-              "fastavro>=0.22.0" "pandas>=0.24.2,<0.25"
+              "fastavro>=0.22.0" "pandas>=0.24.2,<0.25" "hypothesis"
```

Comment:
Why are dependencies for cudf installed in this manual fashion, instead of just doing `conda env update -f ...`?

Reply:
That's a great question 😅, ideally these should be moved to https://github.com/rapidsai/gpuci-build-environment anyway.

Reply:
👍